jenkins-x.yml

```diff
@@ -0,0 +1,15 @@
+buildPack: ml-python-training
+pipelineConfig:
+
+  pipelines:
+    overrides:
+      - pipeline: pullRequest
+        stage: build
+        name: testing
+        step:
+          sh: source /root/.bashrc && python3 -m pip install -r requirements.txt && pytest -v
```

Comment:
no need to specify PR resources as you always develop on master. In face when you try and dev on a branch and create a PR, it fails (I think due to the way Terry set up the buildpack)